Makefile

```diff
@@ -79,5 +81,31 @@ opensbi_jump.bin: opensbi
 	cp opensbi/build/platform/generic/firmware/fw_jump.bin opensbi_jump.bin
 	cp opensbi/build/platform/generic/firmware/fw_jump.elf opensbi_jump.elf
 
+
+u-boot:
+	git clone https://github.com/u-boot/u-boot.git
```

Comment:
Here we will need to fix a tag, otherwise this will clone the latest version and the patch might not apply anymore which would break the build.